xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -565,19 +578,19 @@ This method can be useful if part of a managed object represents a fixed array.
         <Parameter Name="value" Type="System.Byte*" Index="0" FrameworkAlternate="net-6.0;net-7.0" />
       </Parameters>
       <Docs>
-        <param name="value">The pointer to the null-terminated string of bytes.</param>
-        <summary>Creates a new read-only span for a null-terminated UTF8 string.</summary>
+        <param name="value">The pointer to the null-terminated narrow character string.</param>
```

Comment:
I'm not sure talking about characters in describing the arguments is helpful here. I think this should instead be "a sequence of bytes ending with a null value". The remarks can provide potential use cases, but `byte` here is hard for me to accept as "narrow character".

Reply:
I think calling out the use cases is merited, since it talks to why we made the API and what "supported" use cases look like. But your comment about moving that to remarks is well taken.

Reply:
> I think calling out the use cases is merited

Agree. I should have been clearer about my nit. I don't think the notion of "narrow character" should be on a parameter typed `byte`. It makes perfect sense in remarks though and I agree, our intent on use for these kinds of APIs should be shared clearly.